docs/getting_started.md

```diff
@@ -340,7 +340,7 @@ We want to allow access to our application through a reliable named [route](http
     spec_port_target_port: web
     to_name: hello-world
 ```
-The `spec_port_target_port` is the name of the target port on the pods selected by the service this route points to.  The `to_name` is name of the service/target that is being referred.
+The `spec_port_target_port` is the name of the target port on the pods selected by the service this route points to.  The `to_name` is name of the service/target that is being referred.  More information is available in the [openshift_v1_route module](https://github.com/ansible/ansible-kubernetes-modules/blob/master/library/openshift_v1_route.py).
```

Comment:
How about just "The `spec_port_target_port` refers to the name of the target port of the service."